### PR TITLE
GL-908: Change 'green_lanes' to 'check-spimm-eligibility' in URL path.

### PR DESCRIPTION
**Pull request by alexdesi**
### Jira link
https://transformuk.atlassian.net/browse/GL-908

### What?
Change 'green_lanes' to 'check-spimm-eligibility' in URL path.

### Why?
To improve Web usability.

### CODE REVIEW

File: app/services/green_lanes/determine_next_page.rb [hunk 1]
```diff
@@ -65,7 +65,7 @@ def question_unanswered?(cat_exemptions_apply)
     def new_exemptions_path(category, params = {})
       query = params.to_query
 
-      "/green_lanes/applicable_exemptions/new?category=#{category}" + (query.present? ? "&#{query}" : '')
+      "/check-spimm-eligibility/applicable_exemptions/new?category=#{category}" + (query.present? ? "&#{query}" : '')
```
Comment (willfish) [inline app/services/green_lanes/determine_next_page.rb]: Could we use the path helper?

File: config/routes.rb [hunk 2]
```diff
@@ -108,7 +108,7 @@
   resolve('GreenLanes::CategoryAssessmentSearch') { [:category_assessments] }
 
   scope constraints: ->(_req) { TradeTariffFrontend::ServiceChooser.uk? } do
-    namespace :green_lanes do
+    namespace :green_lanes, path: 'check-spimm-eligibility' do
```
Comment (willfish) [inline config/routes.rb]: So 100% of this app uses underscores. Can we just use underscores despite gds guidance?

Comment (alexdesi) [reply config/routes.rb]: I've checked, and they still want dashes because they are easier to type and aligned with all the Gov UK websites.

Comment (willfish) [reply config/routes.rb]: Let's just stick with being consistent with this application and we can review our paths for the entire application at a later date

Comment (willfish) [reply config/routes.rb]: It's not really an option to have a different standard for this one small service

Comment (alexdesi) [reply config/routes.rb]: Ok, at the end we've agreed on _ for now

File: app/services/green_lanes/determine_next_page.rb (hunk 1 above)
Comment (alexdesi) [reply app/services/green_lanes/determine_next_page.rb]: sure, I've also used url_helpers inside the DetermineNextPage,
much better now.

File: spec/requests/green_lanes/applicable_exemptions_controller_spec.rb [hunk 3]
```diff
@@ -38,6 +38,6 @@
     end
 
     it { is_expected.to have_http_status :redirect }
-    it { is_expected.to redirect_to('http://www.example.com/green_lanes/check_your_answers?ans%5B1%5D%5B18fcbb5b75781f8a676bd84dae9c170e%5D%5B%5D=none&ans%5B1%5D%5Ba6b633a7b098132ec45c036d0e14713a%5D%5B%5D=Y997&c1ex=false&category=1&commodity_code=4114109000&country_of_origin=UA&moving_date=2024-05-29') }
+    it { is_expected.to redirect_to('http://www.example.com/check_spimm_eligibility/check_your_answers?ans%5B1%5D%5B18fcbb5b75781f8a676bd84dae9c170e%5D%5B%5D=none&ans%5B1%5D%5Ba6b633a7b098132ec45c036d0e14713a%5D%5B%5D=Y997&c1ex=false&category=1&commodity_code=4114109000&country_of_origin=UA&moving_date=2024-05-29') }
```
Comment (willfish) [inline spec/requests/green_lanes/applicable_exemptions_controller_spec.rb]: I wonder if we want to use path helpers, here, too?

**Review by willfish — DISMISSED** [review]

**Review by willfish — APPROVED** [review]

**Pull request merged**